### PR TITLE
107 readme cleanup

### PR DESCRIPTION
**Pull request by zippy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,23 +110,27 @@ Otherwise, you will need to speak to an admin who can force merge your full chan
...
 
+### App Spec Driven Development
+In adding significant changes and new features to Holochain, we follow a specific test-driven development protocol:
+1. Start by creating a branch in the [app-spec-rust](https://github.com/holochain/app-spec-rust) repository which demonstrates an actual implementation of the use of the new feature in the sample application that lives in that repository, including tests that would pass if the feature were actually implemented here in the holochain-rust repo.
+1. Create a pull request on that branch for the development team to talk about and discuss the suggested change.  The PR triggers Continuous Integration tests which will initially fail, because they try and run the proposed changes against the `develop` branch of this `holochain-rust` repo.
+1. Do any development necessary to on here on `holochain-rust` and `hdk-rust` to implement the feature demonstrated in `app-spec-rust`
```
Comment (lucksus) [inline README.md]: Grammar

File: README.md [hunk 2]
```diff
@@ -91,23 +110,27 @@ Otherwise, you will need to speak to an admin who can force merge your full chan
...
 
+### App Spec Driven Development
+In adding significant changes and new features to Holochain, we follow a specific test-driven development protocol:
+1. Start by creating a branch in the [app-spec-rust](https://github.com/holochain/app-spec-rust) repository which demonstrates an actual implementation of the use of the new feature in the sample application that lives in that repository, including tests that would pass if the feature were actually implemented here in the holochain-rust repo.
+1. Create a pull request on that branch for the development team to talk about and discuss the suggested change.  The PR triggers Continuous Integration tests which will initially fail, because they try and run the proposed changes against the `develop` branch of this `holochain-rust` repo.
+1. Do any development necessary in the `holochain-rust` and `hdk-rust` repos to implement the feature demonstrated in `app-spec-rust`
+1. Finally, when the feature is fully implemented, the CI tests should turn green on `app-spec-rust` and the branch can be merged indicating that that feature.
```
Comment (Connoropolous) [inline README.md]: Sentence trails off. Indicating that that feature... ?

File: README.md [hunk 3]
```diff
@@ -1,37 +1,56 @@
 # Holochain-rust
 
-[![Code Status](https://img.shields.io/badge/Code-Pre_Alpha-red.svg)](https://github.com/holochain/holochain-rust/milestones?direction=asc&sort=completeness&state=all)
+<h1 align="center">
+  <a href="http://holochain.org"><img width="250" src="https://github.com/holochain/org/blob/master/logo/holochain_logo.png?raw=true" alt="holochain logo" /></a>
```
Comment (Connoropolous) [inline README.md]: Looking at this as it displays, I wonder if we could make it smaller? It feels quite big in there, just as a logo

File: README.md (hunk 2 above)
Comment (zippy) [reply README.md]: fixed

**Comment by zippy** [comment]
I did an align right.  I think it looks much better now.

**Pull request merged**